src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -108,7 +108,8 @@ public class PrivilegesEvaluator {
             "indices:admin/shards/search_shards",
             "indices:admin/resolve/index",
             "indices:monitor/settings/get",
-            "indices:monitor/stats"
+            "indices:monitor/stats",
+            "indices:admin/aliases/get"
```

Comment:
It seems to me that the get aliases should also be a DNFOF like behavior - should I keep this in the PR and add a test for it?